pkg/cmd/server/api/validation/validation.go

```diff
@@ -318,3 +318,15 @@ func ValidateExtendedArguments(config api.ExtendedArguments, flagFunc func(*pfla
 
 	return allErrs
 }
+
+func ValidateDomain(domain string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
```

Comment:
defaulting should ensure there is always a value here. if len == 0, consider it an error and return a required error